### PR TITLE
Add secureProxy option to the cookie session

### PR DESCRIPTION
**Pull request by kryswisnaskas**
**Description of change**
Small change to add explicit option to the cookie used for session management.

plus the changes for issue 5

**How to test**

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/257
* closes https://github.com/HHS/Head-Start-TTADP/issues/5

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ app.use(cookieSession({
 
   // Cookie Options. httpOnly is set by default to true for https
   sameSite: 'lax',
+  secureProxy: true,
```
Comment (rahearn) [inline src/app.js:32]: This looks great in dev, but seems to be breaking cookies in my local environment, preventing any logins when `BYPASS_AUTH` is false.

```suggestion
  secureProxy: (process.env.NODE_ENV === 'production'),
```

File: src/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# TTADP Backend
```
Comment (rahearn) [inline src/README.md:1]: Awesome add!

**Review by rahearn — CHANGES_REQUESTED** [review]
Please make this slight tweak to ensure that non-ssl local development still works with HSES login.

**Comment by kryswisnaskas** [comment]
Thanks for finding this. Created  https://github.com/adhocteam/Head-Start-TTADP/pull/174 since I wasn't able to commit the suggested change here.

**Review by rahearn — APPROVED** [review]

**Pull request merged**